Hoping to get this feature implemented.

krv783 opened the pull request:
I did not see any functionality for adding/getting payloads for suggestions. I only saw it for "search". Thanks.

krv783 committed:
Add payload functionality to Suggestions

krv783 committed:
Removed spaces in argument list

stockholmux commented:
Generally, I'm fine with this approach, but the default parameters are only available in Node 6 something and higher I think, so the the package.json `engine` needs to reflect this. That version is fairly old but people run around with _very_ old versions of Node.

krv783 committed:
Rewrite for non-ES6 compatibility

Removed default parameters and tweaked function accordingly to allow compatibility with node core < v6.0
